content/docs/user-guide/interactivity/robotics/importing-robot.md

```diff
@@ -0,0 +1,111 @@
+
+If import fails you will see a message with the output of the ROS 2 XACRO executable.
+
+![Robot Importer](/images/user-guide/gems/ros2/URDF_importer_fail.png)
+
+#### Loading mesh files
+
+6. If the XACRO or URDF file is loaded correctly you will see a table with a list of all mesh files (assets) used in the model. Each row of the table describes one asset, with information divided into the following columns:
+
+- **URDF mesh path** - path extracted from the URDF
+- **Resolved mesh from URDF** - file path of the mesh located in user's filesystem
+- **Product asset** - location of the generated asset (`*.azmodel` or `*.pxmesh`) in the O3DE project assets directory
+- **Source asset** - location of the asset mesh file in the O3DE project assets directory
+- **Type** - the type of the asset, which can be `visual` or `collider`
+
+Besides this information, there is an indication if the asset was imported correctly (green) or if the import failed (red).
+
+Additional information about each asset can be obtained by double-clicking its row. This will open the O3DE Asset Processor showing import details. This information may be especially helpful in case of assets that were not imported correctly.
+
+Click **Next** when finished.
+
+![Robot Importer](/images/user-guide/gems/ros2/URDF_importer_mesh_list.png)
+
+**What to do if there are problems with importing assets?**
+If the Source asset field is marked as failed, it is caused by a URDF importer failure to find a reference to the mesh file in the file system. In such a situation, double-check whether the robot's description package was built and sourced correctly.
+If the Product asset field is marked as failed, it is caused by the Asset Builder failing to build the product asset. Refer to the Asset Processor output to resolve the issue.
+You can do either:
+
+- click **Back**, resolve problems with mesh files and click **Next** to retry import, or
+- proceed by clicking **Next** and finish URDF import. Missing meshes can be added afterward in the O3DE editor.
+
+#### Creating O3DE prefab
+
+7. On the next page:
+
+- Choose a prefab name
+- Indicate (using the checkbox) if robot physics should be based on articulations, or classic rigid bodies and joints structure.
+
+Click **Create Prefab** when finished.
+
+![Robot Importer](/images/user-guide/gems/ros2/URDF_importer_prefab_creation.png)
+
+> Note: the decision whether to use articulations highly depends on the robot structure and simulation requirements. In general, articulations are more stable and computational efficient, but have some limitations. The most important limtation is that articulations natively only support tree-structures - closed loops are not allowed. The "rule of thumb" would be to use articulations in robotic simulation whenever it is possible. However please refer to [this documentation](https://nvidia-omniverse.github.io/PhysX/physx/5.1.3/docs/Articulations.html) for more details.
```

Comment:
This part of the documentation should be adapted to the fact that the release will have PhysX 4 not 5 by default. For this version, O3DE does not support articulations.